### ISSUE TITLE
ProductService

### ISSUE THREAD

**Issue by VZhyrytskiy** [EugeneBuryak/angular2_homework#2]
ProductService добавлен в провайдеры в двух местах app.module.ts, app.component.ts. Так и задумывалось?

**Comment by EugeneBuryak** [comment]
Нет, просто не задумывался над тем как сервис будет использоваться, а после забыл удалить из компоненты после того как зарегал в модуле. 
Спасибо за коментарий.
